components/About/MissionVision.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Container } from "react-bootstrap";
+import Row from "react-bootstrap/Row";
+import BibleVerse from "./BibleVerse";
+import { useTranslation } from "next-i18next";
+
+const MissionVision:React.FC = () => {
+
+    const { t } = useTranslation("about");
+    const missionArray = t("content.mission.content").split("/n")
```

Comment:
I'm a bit wary of having stuff like '/n' in the text portion of the localization json files as Sarah will be making changes there. Could we do something instead like:
`    const missionArray: string[] = t("content.mission.content", { returnObjects: true }); `
and in the json file just have it be an array?
```
"content": [
                "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.",
                "Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum",
                "Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum"
            ]
```